Applications/Utils/SimpleMeshCreation/createMeshElemPropertiesFromASCRaster.cpp

```diff
@@ -35,212 +36,158 @@
-                                              "filename for mesh output");
-    cmd.add( out_mesh_arg );
-
-    TCLAP::ValueArg<bool> refinement_raster_output_arg("",
-                                                       "output-refined-raster",
-                                                       "write refined raster to a new ASC file",
-                                                       false,
-                                                       false,
-                                                       "0");
-    cmd.add( refinement_raster_output_arg );
+        "Generates properties for mesh elements of an input mesh deploying a "
+        "ASC raster file",
+        ' ',
+        "0.1");
+
+    TCLAP::ValueArg<std::string> out_mesh_arg(
+        "o",
+        "out-mesh",
+        "the mesh is stored to a file of this name",
+        false,
+        "",
+        "filename for mesh output");
+    cmd.add(out_mesh_arg);
+
+    TCLAP::ValueArg<bool> refinement_raster_output_arg(
+        "",
+        "output-refined-raster",
+        "write refined raster to a new ASC file",
+        false,
+        false,
+        "0");
+    cmd.add(refinement_raster_output_arg);
 
     TCLAP::ValueArg<unsigned> refinement_arg(
-            "r",
-            "refine",
-            "refinement factor that raises the resolution of the raster data",
-            false,
-            1,
-            "factor (default = 1)");
-    cmd.add( refinement_arg );
-
-    TCLAP::ValueArg<std::string> mapping_arg("",
-                                             "mapping-name",
-                                             "file name of mapping",
-                                             true,
-                                             "",
-                                             "file name");
-    cmd.add( mapping_arg );
+        "r",
+        "refine",
+        "refinement factor that raises the resolution of the raster data",
+        false,
+        1,
+        "factor (default = 1)");
+    cmd.add(refinement_arg);
 
     TCLAP::ValueArg<std::string> raster_arg("",
                                             "raster-file",
                                             "the name of the ASC raster file",
                                             true,
                                             "",
                                             "file name");
-    cmd.add( raster_arg );
+    cmd.add(raster_arg);
+
+    TCLAP::ValueArg<std::string> property_arg(
+        "p",
+        "property-name",
+        "the name of the property the values are stored for",
+        false,
+        "gwn",
+        "property name as string");
+    cmd.add(property_arg);
 
     TCLAP::ValueArg<std::string> mesh_arg("m",
                                           "mesh",
                                           "the mesh is read from this file",
                                           true,
                                           "test.msh",
```

Comment:
Here is a default value, too. That's not from this PR, but maybe you can correct it?